app/lib/server/startup/settings.js

```diff
@@ -243,7 +243,7 @@ settings.addGroup('Accounts', function() {
 			],
 			public: true,
 		});
-		this.add('Accounts_Default_User_Preferences_desktopNotifications', 'mentions', {
+		this.add('Accounts_Default_User_Preferences_desktopNotifications', 'all', {
```

Comment:
Should we really do this for desktop too?  TBH the mobile one didn't annoy me when we first tried this change out.  But the desktop one very quickly made me trigger busy mode to silence them

Reply:
IMHO if we change one we should change both